vectors/nexo/nexo-square.svg

```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" fill="none" viewBox="0 0 48 48">
```

Comment:
Fixed height/width is not allowed as it cannot be converted to PNG easily. I think it can be removed.